ci-operator/step-registry/openshift/e2e/openstack/ipi/openshift-e2e-openstack-ipi-workflow.yaml

```diff
@@ -9,8 +9,7 @@ workflow:
     - chain: ipi-openstack-post
     env:
       OPENSTACK_EXTERNAL_NETWORK: "external"
-      OS_CLOUD: "openstack-cloud"
-      OPENSTACK_COMPUTE_FLAVOR: "m1.s2.xlarge"
+      OPENSTACK_INSTANCE_FLAVOR: "m1.s2.xlarge"
```

Comment:
:) This chances should really be in a different PR, but its not that big of a deal. There is actually not technical reason to change the name of the variable just because the name is strange.